### PR TITLE
Cuts the wand and makes ashdrake armor less like omniarmor

### PR DESCRIPTION
**Pull request by Supermichael777**
The armor was reverted the wand is now going

:cl:
tweak: bio and rad protection removed from ashdrake suit
tweek: cuts the wand of fireball because getting two magic things is slightly op
/:cl:
the conversation

### CODE REVIEW

**Comment by yackemflam** [comment]
Yay

**Comment by WJohn** [comment]
Either nerf the goliath plate upgrades (which can bring explorer armor to 70 melee resist) or keep ashdrake armor as-is. This makes it useless.

**Comment by Ninja137** [comment]
>2500 HP enemy that has multiple ways to oneshot you and heals itself
>Nerfing the drops
It ate almost two full clips from a pulse rifle earlier today. Half damage or not, that's horseshit.

**Comment by yackemflam** [comment]
Though the melee armor value should be higher.

**Comment by ChangelingRain** [comment]
Melee armor shouldn't be reduced, this is a reward for killing an ash drake; it shouldn't be worse than an explorer suit with goliath plates and xenobio fire resist potions.

**Comment by Supermichael777** [comment]
mining has a different balance then station. mining gear coming back to the station has always been a problem because ass-tards can't understand the whole game needs to be balanced. ash drake armor is just for looks anyway. anyway people are finding SOME way to reliably kill this thing because the gear turns up every round.

**Comment by Ninja137** [comment]
Citation needed, fucko.

And if it IS turning up, it's probably because Xenobio can make fucking ash drakes.

**Comment by TechnoAlchemisto** [comment]
The loot is this strong for a reason, you have to be pretty decent to kill a dragon.
I'm not sure what you're trying to say with the slowdown comment, the only non-spaceworthy armor with slowdown anymore is the swat suit and I'm not even sure if that's in cargo anymore.

The armor isn't lavaproof, too, by the way.

This sounds like a stupid nerf based on very little evidence of problems caused by it.

**Comment by Supermichael777** [comment]
it was without question the most powerfull armor in the game that players can get.
revs are going to get the loot because lava loot trivializes pvp.

**Comment by jughu** [comment]
> 30 brute meathook that also stuns for a easy destroy tendril

**Comment by yackemflam** [comment]
>Antag miner comes back to station
>Can't beat them to death
>Can't shoot them to death
>Bombs are much weaker against them
>They fucking can't get stunned because of adrenaline
>Constantly healing
>They have some random shit with them that TRAITORS can TC TRADE FOR usually.
>Not op

**Comment by Ninja137** [comment]
Did this nigga seriously just say that revolutionaries are going to go off-station, kill a fucking Ash Drake with 2,500 HP and multiple instakills, not to mention dealing with all the other shit, just to PVP people?

Do you realize how much fucking work that is, when they could literally just get a spraybottle and a fire extinguisher and fuck anyone over of their choosing?

**Comment by Ninja137** [comment]
Sure, totally can't shoot them. Too bad disablers exist, tasers exist, stun batons exist, flash bangs exist, slipping exists, and they're still vulnerable to bullets just resistant. Not to mention the amount of time spent acquiring all that shit, and how waiting that late in the round to do your shit means that Cargo has more points than it needs to arm up to the teeth, R&D has top-tier equipment, and they're now fighting against Durands.

**Comment by Supermichael777** [comment]
it has hapened more then once
the problem with saying they earned it is that it means your not participating in the round

the revs literaly did nothing else
it was boring as shit

**Comment by Ninja137** [comment]
Citation, mang. And antags can do whatever the hell they want, regardless of participation.

**Comment by TechnoAlchemisto** [comment]
Maybe it's so good because it's one of the hardest items to get in the game?  

Honestly, you can kill them it's not hard, the game is still based on stun combat. I've fought against people in ash drake armor so many times as sec and they go down like everybody else. Lavaland is high risk for high reward. Don't trivialize risk taking by nerfing a reward for something. 

And the "not participating in the round" is boring stupid rhetoric, mining is there to go off station, some people like it some people don't.  Maybe you should actually try to get the armor before you nerf it.

**Comment by Supermichael777** [comment]
repost due to updates

**Comment by Ninja137** [comment]
That does... what beyond prove that there's a circlejerk discord for shitposting?

**Comment by Supermichael777** [comment]
yeah the stun combat argument relies on you already knowing who the antag is.
this is memed with by non antags
if this is official permission to validate mining loot users as security that would be an acceptable compromise.

**Comment by KorPhaeron** [comment]
I'm just curious who Ohlos is now

**Comment by TechnoAlchemisto** [comment]
That discord conversation has confirmed nothing but that you made this PR on behalf of an echochamber.

File: code/modules/mining/lavaland/necropolis_chests.dm [hunk 1]
```diff
@@ -474,7 +474,6 @@
 			new /obj/item/weapon/lava_staff(src)
 		if(3)
 			new /obj/item/weapon/spellbook/oneuse/sacredflame(src)
-			new /obj/item/weapon/gun/magic/wand/fireball(src)
```
Comment (KorPhaeron) [inline code/modules/mining/lavaland/necropolis_chests.dm]: This actually needs to happen though

**Comment by Supermichael777** [comment]
three people and five posts
echochamber
this guy just came in and said it
and i was like i agree
then i made a pr
because im a nice guy

**Comment by Ninja137** [comment]
Yes, because one random fucknugget saying "Hey I've got no real knowledge of [Thing], but I'm going to bitch about it!" is a valid reason for changing the game.

**Comment by Supermichael777** [comment]
that is how the proces works yes

**Comment by Supermichael777** [comment]
welcome to comunity development

**Comment by Ninja137** [comment]
lolno

**Comment by Supermichael777** [comment]
im going to cut the armor change the wand is the real problem

**Comment by PKPenguin321** [comment]
1 guy thinking its bad because he lost a fight to it isn't good to balance around

**Comment by Ninja137** [comment]
Yeeeeah, no. Piss off.

**Comment by yackemflam** [comment]
Did you guys missed that miners also get free anti stun chems?

**Comment by Ninja137** [comment]
u wot m8

**Comment by KorPhaeron** [comment]
Miners have had boxes of stim packs for 3-4 years now

**Comment by An0n3** [comment]
Please don't drag my discord into this bullshit.

**Comment by yackemflam** [comment]
Yes, adding to a job that is already strong at pvp in a game that everyone focuses on pvp is not a very smart move.

**Comment by yackemflam** [comment]
The point is. You guys are giving miners everything BUT stuns and call it balance because they don't have stuns.

**Comment by Ninja137** [comment]
Stimpacks aren't even remotely a problem. It's a difficult set of armor that takes considerable time, skill, and resources to actually get. Attempting to get it runs a very real risk of dying from a wide variety of sources, and there are FAR easier and more effective ways to be an antag than it. Additionally, it's also an obvious set of armor, and that alone gives people a chance to go "Oh shit drake armor, better adapt my tactics"

**Comment by yackemflam** [comment]
And what tatics would you adopt to?

**Comment by Ninja137** [comment]
Get a damn banana peel, spray bottle, disabler, or taser. Get a temperature gun. Or just, y'know, fill their shit with bullets because they aren't immune, just resistant.

If a goddamn clown can take out a full deathsquad while armed with nothing more than a banana and water, then other crewmembers can take out a traitor in drake armor.

**Comment by yackemflam** [comment]
Guess what. stim packs reduces your stun time. They also heal over time due to the legion implant.

File: code/modules/mining/lavaland/necropolis_chests.dm (hunk 1 above)
Comment (lzimann) [inline code/modules/mining/lavaland/necropolis_chests.dm]: what is wrong with it? it's a 4/5 charge wand, not infinite

**Comment by Ninja137** [comment]
So, you have a problem when someone mines enough to get stimpacks, kills a legion and gets it back to the station fast enough to have it successfully implanted, kills an Ash Drake without getting instagibbed/fireballed/dying from claws/burn/whatever else, and makes it back onto the station. Which is going to take them somewhere along the lines of ~20 minutes or so most likely, assuming they find an ash drake in the first place. At which point, unless they murdered all the other miners, the station has top-tier R&D materials, and likely is pumping out traitor gear to hand out like candy. Don't forget the implants the crew has, too, at this point. Or the mechs, bombs, guns, and assorted other tech they'd have to deal with.

This is... worse than Cargo, Chemistry, Botany, R&D, or Xenobio... how, exactly?

**Comment by yackemflam** [comment]
So you're facing a dude who
heals over time
has the best armor in the game obtainable by a player
has a infinite ammo gun that deals decent brute damage
speeds around the station like a speed demon

and if they can use 20 tc
they have no slips
and 18 tc worth of dangerous shit to use

**Comment by XDTM** [comment]
Legion souls do NOT decay when used for surgery, just wanted to clear that up

**Comment by lzimann** [comment]
yackem, let me tell you the secret of life
GET GOOD

**Comment by yackemflam** [comment]
Rnd is also reliant on miners actually bringing stuff in
Rnd also have to use firing pins for their weapons.
Sec has 1 box
RD has the other.
You also don't have security breathing down your neck every time they pass by.

**Comment by yackemflam** [comment]
It takes two point blank shotgun blasts to put you into crit and 10 full laser shots to put you into crit.

**Comment by ChangelingRain** [comment]
Funnily enough, that first one is true even if you're not wearing the armor.

**Comment by yackemflam** [comment]
So, you have a problem when someone mines enough to get stimpacks, kills a legion and gets it back to the station fast enough to have it successfully implanted, kills an Ash Drake without getting instagibbed/fireballed/dying from claws/burn/whatever else, and makes it back onto the station. Which is going to take them somewhere along the lines of ~20 minutes or so most likely, assuming they find an ash drake in the first place. At which point, unless they murdered all the other miners, the station has top-tier R&D materials, and likely is pumping out traitor gear to hand out like candy. Don't forget the implants the crew has, too, at this point. Or the mechs, bombs, guns, and assorted other tech they'd have to deal with.

This is... worse than Cargo, Chemistry, Botany, R&D, or Xenobio... how, exactly?

>Crew
>Implants
>Traitor gear like candy
>Happens more often then the miners coming back as demigods

**Comment by Ninja137** [comment]
The crew regularly gets access so mechs, energy crossbows, BoHs, and other gear from R&D. If they don't, cargo itself can order shitloads of gear. Botany churns out hordes of monsters, has 100 potency Kudzu, firelemons, Deathnettles better than Eswords, access to damn near every chem in the game due to plants, BEES, and other shit. Cargo can get anything that any other department has at all, at basically no effort.

Miners need to waste what's often an entire round, survive a rather tough enemy, and all they get is some kinda good melee armor. It's too bad melee is irrelevant in SS13.

**Comment by ShadowDeath6** [comment]
Don't a lot of mining mobs have armor penetration? Why not just lower armor penetration and adjust armor values accordingly so miners don't have 70 melee resistance on station when it's only really worth like 30 on lavaland?

**Comment by Ninja137** [comment]
Because it isn't even an issue in the first place, and this PR is literally because one dude bitched in some chucklefucks discord, and another chucklefuck tossed up a PR immediately.

**Comment by yackemflam** [comment]
I'm fine with the high melee resistance. But the rest of the armor is on par with swat gear.

**Comment by Ninja137** [comment]
Yes, and? Swat armor is on station. Drake armor requires you to kill a 2,500 HP boss with multiple ways to oneshot you and remove you from the round permanently, and that's on top of even finding it in the first place and getting past the other shit. Oh no, it's strong! Nobody cares, and it's irrelevant. It's a reward for completing something dangerous, and it's appropriate to that.

**Comment by yackemflam** [comment]
You also have other shit that goes WITH your armor.

**Comment by yackemflam** [comment]
The problem isn't the armor itself.
The problem is that miners have so many anti meta shit that they don't even HAVE a way to be stopped.

**Comment by Ninja137** [comment]
Have you tried shooting it until it dies?

**Comment by yackemflam** [comment]
They
cant
die

**Comment by Ninja137** [comment]
I mean, that's objectively wrong, but okay, sure. You keep your fantasies up, man.

**Comment by yackemflam** [comment]
How often have you seen a miner die from guns?

**Comment by Ninja137** [comment]
Well, there'd be... pretty much every time they've tried to fuck with people. Since, y'know, almost nobody ever GETS the combination of stimpacks+drake armor+traitor shit, has time to use it, and doesn't just get gangraped by the station.

**Comment by yackemflam** [comment]
Or just drake armor, stimpacks, no slips, and sleeping carp.
How do you stop them then?

**Comment by Ninja137** [comment]
Memesabre, chainsaw implant, fireaxe, ganging up on them...

**Comment by yackemflam** [comment]
Memesabre - disarm disarm
chainsaw - 70% melee armor laughs at you
Fireaxe - See above
Ganging up - That's a counter for everything ever in spessmen.

**Comment by Ninja137** [comment]
So
What you're saying
So, what you're saying here, Yackem
What you're saying is that there's counters for even an incredibly corner case traitor scenario which won't happen in the first place

It's almost like the armor isn't a problem at all.

**Comment by PKPenguin321** [comment]
>Memesabre - disarm disarm

Yeah exactly it will just dis-arm the miner

**Comment by KorPhaeron** [comment]
I think the stim kit and the infinite regen legion implant are both dumb and I added neither of those to mining

**Comment by yackemflam** [comment]
>Extremely corner case
>Roll miner
>Roll antag
>Kill drake
>Buy carp/noslips
Done

**Comment by yackemflam** [comment]
So you're saying that I can add in a 90% across the board armor that requires you to kill legion and you'd be fine with it?

**Comment by Ninja137** [comment]
That isn't actually what's happening here at all, and legions are piss easy to kill, but hey. You go ahead and twist shit around to be what you want it to be however much you like, man. This is a meme PR because of one chucklefuck, and you're taking it way too seriously.

**Comment by Hatfish** [comment]
People kill miners all the time, even when they've got drake armor, stimpacks or legion implants.
This is literally a case of you needing to git gud.

**Comment by KorPhaeron** [comment]
Ironically, the last round I just watched involved Yackemflam killing a miner in drake armour, then getting killed in turn by a taser after they took the lava staff

You can sleeping carp+hulk+chems/virus as well for completely stun proof bullet proof regenerating health. Complete projectile immunity can get silly, surprise.

**Comment by yackemflam** [comment]
I was an op that round.

**Comment by yackemflam** [comment]
And the guy can't aim at all with a fucking lavastaff.

**Comment by XDTM** [comment]
Did everyone forget stunbatons

**Comment by An0n3** [comment]
watching Ninja and Yackem get in a spergwar is one of the most shameful things Ive ever seen in this community and I regret posting on this PR becausr of it

**Comment by Kromgar** [comment]
Miners get dunked all the time. Stim packs dont make you immune to stuns.

**Comment by kevinz000** [comment]
pr made by someone who suggests giving the HOS a hitscan taser.

File: code/modules/clothing/suits/cloaks.dm [hunk 2]
```diff
@@ -66,7 +66,7 @@
 	icon_state = "dragon"
 	desc = "A suit of armour fashioned from the remains of an ash drake. "
 	allowed = list(/obj/item/device/flashlight,/obj/item/weapon/tank/internals, /obj/item/weapon/resonator, /obj/item/device/mining_scanner, /obj/item/device/t_scanner/adv_mining_scanner, /obj/item/weapon/gun/energy/kinetic_accelerator, /obj/item/weapon/pickaxe, /obj/item/weapon/twohanded/spear)
-	armor = list(melee = 70, bullet = 30, laser = 50, energy = 40, bomb = 70, bio = 60, rad = 50)
+	armor = list(melee = 70, bullet = 30, laser = 50, energy = 40, bomb = 70, bio = 0, rad = 0)
```
Comment (Fox-McCloud) [inline code/modules/clothing/suits/cloaks.dm]: >reducing bio-armor as if it does absolutely anything at all.

Comment (KorPhaeron) [inline code/modules/clothing/suits/cloaks.dm]: I was gonna post this but I couldn't be bothered to find the documentation for the virus formula. Pretty sure its either 100 and it blocks or else you get infected.

Comment (Fox-McCloud) [inline code/modules/clothing/suits/cloaks.dm]: viruses are based off of `permeability_coefficient`...bio armor is literally a useless stat.

Comment (KorPhaeron) [inline code/modules/clothing/suits/cloaks.dm]: Does it block the decloner gun at least?

Comment (Fox-McCloud) [inline code/modules/clothing/suits/cloaks.dm]: Decloner inherits the `"energy"` flag from its parent, so, no.

even reagent protection goes off of permeability coefficient. 

I kinda get the idea that someone intended to get rid of one or the other, at some point, but never got around to doing it.

**Comment by Cheridan** [comment]
universally disliked

also >83 comments in 13 hours

jesus

**Pull request closed without merging**